book/09-git-and-other-scms/sections/import-svn.asc

```diff
@@ -31,7 +31,14 @@ That generates the log output in XML format, then keeps only the lines with auth
 Then, redirect that output into your `users.txt` file so you can add the equivalent Git user data next to each entry.
 
 You can provide this file to `git svn` to help it map the author data more accurately.
-You can also tell `git svn` not to include the metadata that Subversion normally imports, by passing `--no-metadata` to the `clone` or `init` command (though if you want to keep the synchronisation-metadata, feel free to omit this parameter).
+You can also tell `git svn` not to include the metadata that Subversion normally imports, by passing `--no-metadata` to the `clone` or `init` command.
+The metadata includes a `git-svn-id` inside each commit message that Git will generate during import. This can bloat your Git log and might make it a bit unclear.
```

Comment:
Two thoughts:

1. We like to format the text in this repo to have one sentence per line of source. This makes diffs _much_ more readable when changes need to be made.
1. Let's kill line 40, and change the second sentence here to read `` If you don't want this extra noise in your commit log, feel free to omit the `--no-metadata` parameter. ``

Reply:
I saw you made a commit, but it didn't fix what I asked about. You added a blank line before this line, making it a paragraph; I was asking for you to split this line at the period. And you didn't remove line 40, you just edited it.

Reply:
Now I'm completely confused.

Reply:
1. This line of code contains two sentences. "The metadata includes…" and "This can bloat…". Please add a line break in between to help future contributors and PR reviewers.
1. 🤔 would "content" be better than "noise?"